docs/src/main/znai/auth/auth-and-impersonation.md

```diff
@@ -0,0 +1,79 @@
+Liftwizard supports the dynamic configuration of [dropwizard-auth](https://www.dropwizard.io/en/stable/manual/auth.html) in `config.yml` and the configuration of impersonation in tests.
+
+Add a dependency on `liftwizard-bundle-auth-filter` and add `AuthFilterBundle` to the registered bundles.
+
+Modify the application's configuration class to implement `AuthFilterFactoryProvider` if it does not already extend `AbstractLiftwizardConfiguration`.
```

Comment:
Maybe also note here that this needs the dependency on `liftwizard-config-auth-filter-header`?